cartographer/mapping/collated_trajectory_builder.cc

```diff
@@ -31,8 +31,7 @@ constexpr double kSensorDataRatesLoggingPeriodSeconds = 15.;
 CollatedTrajectoryBuilder::CollatedTrajectoryBuilder(
     sensor::Collator* const sensor_collator, const int trajectory_id,
     const std::unordered_set<std::string>& expected_sensor_ids,
-    std::unique_ptr<GlobalTrajectoryBuilderInterface>
-        wrapped_trajectory_builder)
+    std::unique_ptr<TrajectoryBuilderInterface> wrapped_trajectory_builder)
```

Comment:
consistency: usually we don't name unique_ptrs "wrapped_"
rename to: global_trajectory_builder, also the member

Reply:
So "wrapped" does not refer to the fact that it is wrapped in a unique_ptr but rather that it is the `CollatedTrajectoryBuilder` that wraps it. I wouldn't call it `global_trajectory_builder` because in principle a `CollatedTrajectoryBuilder` can wrap any implementation of `TrajectoryBuilderInterface`.